docs/views/layout.html

```diff
@@ -49,13 +49,19 @@
 {% endblock %}
 
 {% block footer_support_links %}
-  {% if useAutoStoreData %}
-    <ul>
-      <li>
-        <a href="/prototype-admin/clear-data">Clear data</a>
-      </li>
-    </ul>
-  {% endif %}
+  <ul class="govuk-!-mb-r3 app-!-db">
```

Comment:
This looks like a govuk-frontend class - will this work in the kit as it stands?